packages/destination-actions/src/destinations/intercom/groupIdentifyContact/index.ts

```diff
@@ -0,0 +1,142 @@
+import { ActionDefinition, ModifiedResponse, RequestClient, RetryableError } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+interface IntercomCreateCompanyData {
+  id: string
+}
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Group Identify Contact',
```

Comment:
```suggestion
  title: 'Identify Company',
```
Open to other suggestions. Intercom doesn't use the language "group identify" so I think that would be a confusing way to explain this action.

Reply:
sounds good to me!